dashboard/15-final/app/api/auth/[...nextauth]/route.ts

```diff
@@ -0,0 +1,53 @@
+import NextAuth, { NextAuthOptions } from 'next-auth';
+import CredentialsProvider from 'next-auth/providers/credentials';
+import bcrypt from 'bcrypt';
+import { fetchUser } from '../../../lib/data';
+import { User } from '@/app/lib/definitions';
+
+const authOptions: NextAuthOptions = {
```

Comment:
I suggest moving authOptions to a top-level `auth.ts` file, which will be the new recommendation when v5 comes out, so this will help migration. People already put this config into different files.

Check out https://github.com/nextauthjs/next-auth/blob/main/apps/examples/nextjs/auth.ts